src/file/file.repository.ts

```diff
@@ -13,34 +13,66 @@ const sort = {
 };
 
 export default class FilesRepository {
+  /**
+   * Adds a given file to the DB.
+   * @param file is the file to be added to the DB
```

Comment:
Missing `-` after 'file'.

Reply:
Rename class `FilesRepository` name to `FileRepository`.
Document the class' purpose.